common.go

```diff
@@ -67,8 +67,8 @@ type FieldSort struct {
 	D Dir
 }
 
-// String returns a representation of Sort compatible with the format of mgo.
-func (s Sort) String() string {
+// Mgo returns a representation of Sort compatible with the format of mgo.
+func (s Sort) MgoFormat() []string {
```

Comment:
That's leftovers because I changed my mind about the name and then forgot to change the comment :-1: :doh: I'll rename it to `List`.